internal/remote/user.go

```diff
@@ -57,16 +59,14 @@ func (user *User) GetUpdates() <-chan imap.Update {
 }
 
 func (user *User) Close(ctx context.Context) error {
+	close(user.forwardQuitCh)
+	user.forwardWG.Wait()
+	user.updatesCh = nil
```

Comment:
this scares me a bit, because here we set `user.updatesCh` to nil, and below (in `forward(...)`) we close it. Closing a nil channel leads to a panic.

i'm trying to follow the system of waitgroups and defers to see if we're guaranteed not to close after setting it to nil but im getting a bit confused haha -- i'm thinking we don't even need to set it to nil at all? kind of a pointless operation, no?

Reply:
It is safe a this point. But removed as it is unnecessary.